index.css

```diff
@@ -78,8 +88,372 @@
     margin-left: 10%;
   }
 
+  .works-centered img {
+    width: 100%;
+  }
+
+  .attach-icon img {
+    float: left;
+    width: 5%;
+    height: 5%;
+    margin-right: 2%;
+  }
+
   .social a img {
     width: 19.99px;
     height: 19.96px;
   }
+
+  .extra {
+    margin-left: 6%;
+    margin-right: 6%;
+  }
+
+  .works-centered {
+    display: inline-flex;
+    flex-direction: column;
+    margin-top: 50%;
+  }
+
+  .works-title {
+    font-family: 'Crete Round', serif;
+    font-size: 400%;
+    margin-top: -10%;
+    text-align: center;
+  }
+
+  .works-centered hr {
+    list-style: solid;
+    width: 100%;
+    color: #212121;
+    margin-top: 4%;
+    margin-bottom: 27%;
+  }
+
+  .works-centered .multi-stories {
+    font-family: 'Crete Round', serif;
+    font-size: 350%;
+    text-align: left;
+  }
+
+  .works-centered p {
+    font-family: 'Inter', sans-serif;
+    font-size: 200%;
+    text-align: left;
+    margin-top: -3%;
+    padding-bottom: 15px;
+  }
+
+  .works-centered ul {
+    display: flex;
+    flex-direction: row;
+    align-items: left;
+    list-style: none;
+    margin-bottom: 60px;
+  }
+
+  .works-centered a {
+    text-decoration: none;
+    border: solid #8b8282d3;
+    padding: 30px;
+    font-family: 'Inter', sans-serif;
+    font-size: 200%;
+    margin: 13px;
+    color: #1c1a19;
+  }
+
+  .list-x a {
+    text-decoration: none;
+    border: solid #8b8282d3;
+    padding: 18px;
+    font-family: 'Inter', sans-serif;
+    font-size: 200%;
+    color: #fff;
+    background-color: rgba(255, 255, 255, 0.24);
+  }
+
+  .lists a {
+    text-decoration: none;
+    font-size: 250%;
+  }
+
+  .project-list a:hover {
+    background-color: #ff6b00;
+  }
+
+  .project-list .start-list {
+    margin-left: -23%;
+  }
+
+  .project-list .second-list {
+    margin-left: -20%;
+  }
+
+  .project-list .third-list {
+    margin-left: -8%;
+  }
+
+  .works-centered button {
+    display: flex;
+    align-items: left;
+    font-size: 200%;
+    margin-left: 0%;
+    margin-right: 60%;
+    padding: 30px;
+    cursor: pointer;
+    border: none;
+    background-color: #ff6b00;
+    font-weight: bold;
+    color: white;
+  }
+
+  .print-data button {
+    border: none;
+    background-color: #ff6b00;
+    cursor: pointer;
+    font-weight: bold;
+    color: white;
+    font-size: 200%;
+  }
+
+  .interesting button {
+    margin-left: 30%;
+    margin-right: 30%;
+    border: none;
+    cursor: pointer;
+    background-color: #ff6b00;
+    font-weight: bold;
+    color: white;
+    font-size: 200%;
+    padding: 4% 4% 4% 4%;
+    margin-bottom: 22%;
```

Comment:
- [ ] Kindly use pixels instead percentages for the font size and margins to be similar to the Figma design.